Fix some bugs

daniddelrio opened the pull request:
Please check the ff:
1. Tutee settings should be accessible now
2. "View code" should now be removed from tutor home and from Upcoming sessions of tutee
3. Security codes of the tutee should not be stored anymore
4. Upcoming pages should have a subheader now
5. You should have the ability to upload display pictures

LienLeinad commented:
Getting an error for validation when registering a tutor account
This is what comes up in the command prompt
![image](https://user-images.githubusercontent.com/33220418/77840920-e94bb800-71be-11ea-927e-de761fd6bda2.png)

when I create a tutor acct it keeps giving me an error saying the fields are invalid
other than that everything else seems to work, although I haven't checked the security code thing since I can't make a tutor acct

daniddelrio commented:
Fixed, thanks Neil. I also forgot to remove the Sec Code field in the tutee registration so I just removed it now. 

Also, just a heads up, I haven't added the uploading of a display picture to the registration, so you can only upload a display picture on your Settings page once you've already created an account. I plan to add the uploading feature during registration once the validation for registration is fixed; I think that's your ticket.
